send-email.js

```diff
@@ -0,0 +1,60 @@
+import nodemailer from 'nodemailer';
+import fs from 'fs';
+import dotenv from 'dotenv-safe';
+import { getAllRecords } from './lib/airtable-prod/airtable';
+import { Tables } from './lib/airtable-prod/schema';
+
+dotenv.config();
+
+const sendEmail = async () => {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASS,
+    },
+  });
+
+  const emailList = await getAllRecords(Tables.Emails)
```

Comment:
you could probably do away with the .then block if you are using await.

Reply:
removed await